### ISSUE TITLE
Edit submission review page

### ISSUE THREAD

**Issue by rsgonzal** [Iridescent-CM/technovation-app#3229]
The page students get to to review their submisison materials is missing fields. This is also what they see to view their submitted materieals.

Mockups [here](https://app.mural.co/t/technovationgirls4378/m/technovationgirls4378/1641856879995/a4c8a43afaaf7b1cc34e20b1768eb1bbd3b9d6f3?sender=u9adab98cc36af2c4665e9477)

**Comment by dboyer** [comment]
I noticed a line about regional pitch events too when I submitted my app. We'll need to remove that.

**Comment by shaunxp20** [comment]
A version of the submission review page is on QA, that we can review and iterate on. 😄 

Right now this new page will only be shown after a submission has been completed/submitted, and only shown from My Dashboard. This was the easiest way to get it integrated into the submission flow - there are quite a few states/flows a submission can be in depending on season toggles and the state of the submission. We can incorporate this new design into these flows of course, this was basically just getting a rough draft on QA for us to look at. 😄

**Comment by rsgonzal** [comment]
![screencapture-technovation-qa-herokuapp-student-published-team-submissions-no-name-yet-melan45-2022-02-24-12_22_44 (1).png](https://images.zenhubusercontent.com/60527169cde2772242d0de95/33af42c0-8285-408f-9632-c412685e127d)

**Comment by shaunxp20** [comment]
Just FYI, the submission review page is now a part of the submission flow - on the last page before submitting (the page shown in https://github.com/Iridescent-CM/technovation-app/issues/3229#issuecomment-1051070557), this is on QA now ready for a looksie. 😄

**Comment by rsgonzal** [comment]
Woooo. it's looking really good. only thing i noticed @shaunxp20 is the entrepreneurship section is showing up for beginner. the link basically just brings you back to the same page because beginners can't upload

![Screen Shot 2022-03-08 at 9.25.33 AM.png](https://images.zenhubusercontent.com/60527169cde2772242d0de95/7c31dbad-8b1b-46bb-b070-bdc496575da0)

**Comment by shaunxp20** [comment]
> Woooo. it's looking really good. only thing i noticed @shaunxp20 is the entrepreneurship section is showing up for beginner. the link basically just brings you back to the same page because beginners can't upload

Good catch, and thanks for looking this over! 🥳 I work on getting this updated now! 😄

**Comment by dboyer** [comment]
Looks great! Only thing I noticed is that if I've selected a Thunkable project while submitting, the review page has the option to "download the technical work." When I click that, it opens the same review page in a new tab.

I realize there isn't any technical work to download since the submission just includes a link to Thunkable so this may be the intended behavior. Noting it here in case something else should be happening.
